tracim/tracim/lib/content.py

```diff
@@ -450,6 +452,34 @@ def get_one_revision(self, revision_id: int = None) -> ContentRevisionRO:
 
         return revision
 
+    # def get_one_revision_file(self, revision_id: int = None):
```

Comment:
Commenting a full function must always come with some explanation : 
- first case is "the code must be removed" => remove it
- otherwise, there is a reason for keeping the code commented in sources => explain why

Note : comments to be formatted like this:
```
# TODO - P.A - 2017-07-01 - <short resume here>
# <extra explanation here
# <extra explanation may be on several lines>
```

Note : the first word of first line must be : HACK / TRICKY / FIXME / TODO / INFO